scratch-debugger/debugger.yaml

```diff
@@ -0,0 +1,31 @@
+# Debugger pod used for installing busybox in the target pod via ./debug.sh
+apiVersion: v1
+kind: Pod
+metadata:
+  name:        ARG_DEBUGGER
+  namespace:   ARG_NAMESPACE
+spec:
+  nodeName:    ARG_NODENAME
+  containers:
+    - name:    debugger
+      image:   busybox
```

Comment:
Added ARCH. For these purposes, I'd rather use `latest` than pin to a specific version. I can add a parameter for it if you think it's necessary...